sale_order_type/README.rst

```diff
@@ -0,0 +1,43 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+Sale Order Types
+================
+
+This module adds a typology for the sale orders.
```

Comment:
Explain that you can select a journal in the order type and this journal is selected when invoicing, and each partner can have a default order type.